.changeset/curly-planets-laugh.md

```diff
@@ -0,0 +1,5 @@
+---
+'@openfn/language-http': patch
+---
+
+Add contentType option for automatic `Content-Type` header allocation.
```

Comment:
Something suspicious about this changeset.

We can hopefully just delete it, but I fear you've branched from the wrong place